app/assets/javascripts/adoption_app_home_auth.js

```diff
@@ -0,0 +1,11 @@
+$(document).ready(function () {
+    $(".own-home-auth").hide();
+    $(".residence-auth-pop").change(function () {
+        if (this.value == 'own') {
+            $(".own-home-auth").show();
+        }
+        else if (this.value == 'rent') {
+            $(".own-home-auth").hide();
+        }
```

Comment:
Well, technically it's a radio button so the only two options "should" be a rent or own.  But logically an else hide is more accurate since we'd always want the field hidden if the value was other than own.  I'll toss the update in.